package tacs.grupo_4.exceptions;

public class UsuarioNotFoundException extends RuntimeException {
}
